tests/multipleBackend/objectPut.js

```diff
@@ -157,6 +157,10 @@ describeSkipIfE2E('objectPutAPI with multiple backends', function testSuite() {
                     checkPut(testCase);
                     next();
                 }),
+                // Increase the probability of the first request having released
+                // the socket, so that it can be reused for the next request.
+                // This tests how HTTP connection reuse behaves.
+                next => setTimeout(next, 10),
```

Comment:
It's not about closing the socket, but having the first request not use the socket anymore, so that the second can use it. It's pure js logic, not OS. This very small timeout seems to leave the time for the code responsible for this to be scheduled and release the socket, therefore making socket reuse much more frequent (checked by looking at nginx logs). Otherwise, the second request often uses a new socket instead of the existing one, defeating the purpose of the test.